test/unittests/imperative.test.js

```diff
@@ -1,22 +1,18 @@
+
```

Comment:
I think it'd be better to just have 2 files here for each of the tests.

Also, why the need for a separate file in fixtures when you can just call the same file in a child process just with a specific `env` variable set (and then check `if (process.env.worker) ...`)?

Reply:
I think that abstracting the test itself from forking the child process is a better idea, than do everything in one file.